src/dataSources/WikibaseActionApi.js

```diff
@@ -8,8 +8,13 @@ module.exports = class WikibaseActionApi extends RESTDataSource {
     this.baseURL = baseUrl;
   }
 
-  async getEntity(id) {
-    return this.getEntitiesLoader.load(id);
+  async getEntity(queryValue, queryLang) {
```

Comment:
I think this method should also stay as it is and there should be an additional `searchEntities` method that does the request to `wbsearchentities`.